spec/forms/idv/finance_form_spec.rb

```diff
@@ -5,21 +5,21 @@
 
   describe 'presence validations' do
     it 'is invalid when required attributes are not present' do
-      valid_params = { finance_type: :ccn, finance_account: '12345678' }
+      valid_params = { finance_type: :ccn, ccn: '12345678' }
 
-      [:finance_type, :finance_account].each do |attr|
+      [:finance_type, :ccn].each do |attr|
         subject.submit(valid_params.merge(attr => nil))
 
         expect(subject).to_not be_valid
         expect(subject.errors.full_messages.first).
-          to eq "#{attr.to_s.humanize} #{t('errors.messages.blank')}"
+          to eq "Ccn #{t('errors.messages.blank')}"
```

Comment:
When it's rendered, it doesn't have `Ccn` (I think this is a result of squishing a hash-like object into an array)